### PR TITLE
Motor controller update

### PR DESCRIPTION
**Pull request by Obbay2**
Pull request to put the Talon PWM fixes into Scarlet.

### CODE REVIEW

File: Components/Motors/TalonMC.cs [hunk 1]
```diff
@@ -1,16 +1,16 @@
-﻿using System;
+using System;
 using Scarlet.Filters;
 using Scarlet.IO;
 using Scarlet.Utilities;
 using System.Threading;
 
-namespace Scarlet.Components.Motors
+namespace Talon
```
Comment (CaiB) [inline Components/Motors/TalonMC.cs]: Can you change this back? This will break things if left as is

File: Components/Motors/TalonMC.cs [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 {
-    public class TalonMC : IMotor
+    public class TalonMC : Scarlet.Components.IMotor
```
Comment (CaiB) [inline Components/Motors/TalonMC.cs]: Remove unnecessary definition here

File: Components/Motors/TalonMC.cs [hunk 3]
```diff
@@ -25,8 +25,8 @@ public TalonMC(IPWMOutput PWMOut, float MaxSpeed, IFilter<float> SpeedFilter = n
             this.PWMOut = PWMOut;
             this.MaxSpeed = Math.Abs(MaxSpeed);
             this.Filter = SpeedFilter;
-            this.SetSpeedDirectly(0.0f);
             this.PWMOut.SetFrequency(333);
+            this.SetSpeedDirectly(0.0f);
```
Comment (CaiB) [inline Components/Motors/TalonMC.cs]: Did you mean to move this?

**Comment by Baldstrom** [comment]
I already fixed these bugs on master

**Comment by Obbay2** [comment]
Closed due to already being fixed!

**Pull request closed without merging**